contracts/Conductor.sol

```diff
@@ -31,7 +45,24 @@ contract Conductor is IConductor, Ownable {
 	) external returns (address, address) {
 		Registrar short = new Registrar(roleChecker, shortName, shortSymbol, version, registrarType);
 		Registrar long = new Registrar(roleChecker, longName, longSymbol, version, registrarType);
-        emit Conducted(_id, address(short), address(long));
+
+    emit Conducted(
```

Comment:
I think one event is better than two events because we are reading the events and it is better to have one event per registrar.

Reply:
Shouldn't we need an Event for the single deployment as well? Else, indexers wouldn't be informed of it. In that case, it makes sense to split the Conducted event by registrarType (like I've proposed) so single deployments can still adhere to the same format.

Reply:
Yes. It has been fixed.